apps/paf/fields/index.js

```diff
@@ -1413,7 +1413,7 @@ module.exports = {
       field: 'are-you-eighteen',
       value: 'yes'
     },
-    validate: [{ type: 'maxlength', arguments: 20 }]
+    validate: ['required', 'internationalPhoneNumber', { type: 'maxlength', arguments: 30 }]
```

Comment:
Is this ok at 30 or should it be 200 like the others? I'd say best to keep them all equal

Reply:
Yes i changed it since other where 200.